🌐 [i18n-KO] Translated `tutorial/proprecssing.mdx` to Korean

sim-so opened the pull request:
# What does this PR do?

<!--
Congratulations! You've made it this far! You're not quite done yet though.

Once merged, your PR is going to appear in the release notes with the title you set, so make sure it's a great title that fully reflects the extent of your awesome contribution.

Then, please replace this with a description of the change and which issue is fixed (if applicable). Please also include relevant motivation and context. List any dependencies (if any) that are required for this change.

Once you're done, someone will review your PR shortly (see the section "Who can review?" below to tag some potential reviewers). They may suggest changes to make the code even better. If no one reviewed your PR after a week has passed, don't hesitate to post a new comment @-mentioning the same persons---sometimes notifications get lost.
-->

<!-- Remove if not applicable -->

Translated ~partially~ the documentation `tutorial/proprecessing.mdx` to Korean.
- [x] Proprecessing 전처리
  - [x] Natural Language Processing 자연어처리
    - [x] Pad 패딩
    - [x] Trancation 생략
    - [x] Build Tensor 텐서 만들기
  - [x] Audio 오디오
  - [x] Computer Vision 컴퓨터 비전
    - [x] Pad 패딩
  - [x] Multimodal 멀티모달

Thank you in advance for your review!

Part of #20179 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?

Team PsedoLab, could you review this PR?

@wonhyeongseo @0525hhgus @kihoon71 @gabrielwithappy, @HanNayeoniee, @jungnerd 

<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @

 If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
 Please tag fewer than 3 people.

Models:

- text models: @ArthurZucker and @younesbelkada
- vision models: @amyeroberts
- speech models: @sanchit-gandhi
- graph models: @clefourrier

Library:

- flax: @sanchit-gandhi
- generate: @gante
- pipelines: @Narsil
- tensorflow: @gante and @Rocketknight1
- tokenizers: @ArthurZucker
- trainer: @sgugger

Integrations:

- deepspeed: HF Trainer: @stas00, Accelerate: @pacman100
- ray/raytune: @richardliaw, @amogkam

Documentation: @sgugger, @stevhliu and @MKhalusova

HF projects:

- accelerate: [different repo](https://github.com/huggingface/accelerate)
- datasets: [different repo](https://github.com/huggingface/datasets)
- diffusers: [different repo](https://github.com/huggingface/diffusers)
- rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)

Maintained examples (not research project or legacy):

- Flax: @sanchit-gandhi
- PyTorch: @sgugger
- TensorFlow: @Rocketknight1

 -->

sim-so committed:
add ko preprocessing

sim-so committed:
translate preprocessing.mdx to korean

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_22578). All of your documentation changes will be reflected on that endpoint.

gabrielwithappy commented:
@sim-so keep it up 👍 :-)

gabrielwithappy commented on docs/source/ko/preprocessing.mdx:
````diff
@@ -0,0 +1,525 @@
+'[CLS] Do not meddle in the affairs of wizards, for they are subtle and quick to anger. [SEP]'
+```
+
+보시다시피 토크나이저는 두 개의 특수한 토큰 - `CLS`와 `SEP` (분류토큰과 구분토큰) - 을 문장에 추가했습니다. 
````
I think we can revove redundent word "보시다시피"
How about below?
토크나이저는 두 개의 특수한 토큰 - `CLS`와 `SEP` (분류토큰과 구분토큰) - 을 문장에 추가했습니다.

wonhyeongseo replied:
I agree with @gabrielhappy. We could also move the non-essential clause in front of the noun inside parentheses. Like so:

sim-so replied:
Thank you for your reviews! :smile:

I made two revisions based on your feedback as below:
```
1. 토크나이저가 `CLS`(분류 토큰)와 `SEP`(구분 토큰)라는 두 개의 특수한 토큰을 문장에 추가했습니다.
2. 토크나이저가 두 개의 특수한 토큰(분류 토큰 `CLS`와 구분 토큰 `SEP`)을 문장에 추가했습니다.

```
By the way, is it good enough: '분류 토큰' and '구분 토큰' for 'classifier' and 'separator`?
Would it be more helpful if I add original words to indicate where the name of the tokens came from?
Like below:
`1. 토크나이저가 `CLS`(분류 토큰, classifier)와 `SEP`(구분 토큰, separator)라는 두 개의 특수한 토큰을 문장에 추가했습니다.`

wonhyeongseo replied:
I would add in the original words since abbreviations can be harder to understand for people using the tutorial.
Since we discussed `-er`/`-or` suffixes today, how about replacing '토큰' with '-자' like:

> 토크나이저가 `CLS`(분류자; classifier)와 `SEP`(구분자; separator)라는 두 개의 특수한 토큰을 문장에 추가했습니다.

Thank you for your revisions and hard work! 👍

gabrielwithappy reviewed (changes requested):
Please check my review :-)

sim-so committed:
translate preprocessing.mdx

sim-so committed:
Merge branch 'main' into ko-preprocessing

sim-so commented:
Some updates!
- Newly translated two parts of this documentation: `Build Tensors` and `Audio`
- Revised the sentence with the feedback as below:
`토크나이저가 두 개의 특수한 토큰(분류 토큰 CLS와 구분 토큰 SEP)을 문장에 추가했습니다.`
- Translated and revised all `feature extractor` to `특징 추출기` based on TTA.
I am going to finish it by this Sunday. Thank you all in advance! :smile:

0525hhgus commented on docs/source/ko/preprocessing.mdx:
````diff
@@ -0,0 +1,527 @@
+>>> feature_extractor = AutoFeatureExtractor.from_pretrained("facebook/wav2vec2-base")
+```
+
+오디오 `array`를 특징 추출기에 전달하세요. 또한, 특징 추출기에 `sampling_rate` 인자를 추가하여 발생할 수 있는 silent errors(즉각적인 오류 메시지가 발생하지 않는 오류)를 더 잘 디버깅하는 것을 권장합니다.
````
I checked the meaning you explained in "silent errors(즉각적인 오류 메시지가 발생하지 않는 오류)".

If you translate it, it would be good to use "잠복 오류" and "잠재 오류" in similar meanings.
Although it is a different field, there are cases where "silent" is used as "잠재된" or "잠복된" in the medical field.

It's my personal opinion, so please refer to it 😄

sim-so replied:
Thank you for giving opinion!
Follwing the opnions from you and other team members, I edited the word as "조용한 오류(silent errors)".

sim-so committed:
Update preprocessing.mdx

Fixed the line 273 as below:
또한, 특징 추출기에 `sampling_rate` 인자를 추가하여 발생할 수 있는 조용한 오류(silent errors)를 더 잘 디버깅하는 것을 권장합니다.

sim-so committed:
translate Image part

sim-so committed:
translated preprocess.mdx

sim-so commented:
I translated all of this document.
Thank you in advance for your review! 😉

0525hhgus commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -468,23 +468,25 @@ from [`DetrImageProcessor`] and define a custom `collate_fn` to batch images tog
 ## 멀티모달[[multimodal]]
 
-For tasks involving multimodal inputs, you'll need a [processor](main_classes/processors) to prepare your dataset for the model. A processor couples together two processing objects such as as tokenizer and feature extractor.
+다중 모달 입력이 필요한 작업의 경우, 모델에 데이터셋을 준비하기 위한 [프로세서](main_classes/processors)가 필요합니다. 
```
'multimodal' is translated as '멀티모달' in the section title, but appears as '다중 모달' in the following sentence.
If it's a translation of a word that hasn't been confirmed yet, please check it slowly.

sim-so replied:
```suggestion
멀티모달 입력이 필요한 작업의 경우, 모델에 데이터셋을 준비하기 위한 [프로세서](main_classes/processors)가 필요합니다. 
```
I translate "멀티모달" for this time. If another word gets confirmed, I will change it later.

0525hhgus reviewed (commented):
Thank you for the good translation 👍 
Please check the comments below 😃

wonhyeongseo commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -0,0 +1,535 @@
+
+[[open-in-colab]]
+
+모델을 학습하려면 데이터셋을 모델에 맞는 입력 형식으로 전처리 해야 합니다. 데이터가 텍스트, 이미지 또는 오디오인지 여부에 관계없이 데이터를 텐서 배치로 변환하고 조립할 필요가 있습니다. 🤗 Transformers는 모델에 대한 데이터를 준비하는 데 도움이 되는 일련의 전처리 클래스를 제공합니다. 이 튜토리얼에서는 다음 내용을 배울 수 있습니다:
```
```suggestion
모델을 학습하려면 데이터셋을 모델에 맞는 입력 형식으로 전처리 해야 합니다. 데이터가 텍스트, 이미지 또는 오디오인지 여부에 관계없이 데이터를 텐서 배치로 변환하고 조립할 필요가 있습니다. 🤗 Transformers는 모델에 대한 데이터를 준비하는 데 도움이 되는 일련의 전처리 클래스를 제공합니다. 이 튜토리얼에서 배울 내용은:
```

sim-so replied:
```suggestion
모델을 학습하려면 데이터셋을 모델에 맞는 입력 형식으로 전처리 해야 합니다. 데이터가 텍스트, 이미지 또는 오디오인지 여부에 관계없이 데이터를 텐서 배치로 변환하고 조립할 필요가 있습니다. 🤗 Transformers는 모델에 대한 데이터를 준비하는 데 도움이 되는 일련의 전처리 클래스를 제공합니다. 이 튜토리얼에서는 다음 내용을 배울 수 있습니다.
```
저는 문장이 완결되는 것이 더 좋은 것 같아서 `:` 대신 `.` 으로 바꾸는 것을 제안드립니다. 어떻게 생각하세요?

wonhyeongseo commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -0,0 +1,535 @@
+
+</Tip>
+
+시작하기 전에 🤗 Datasets를 설치하여 실험에 사용할 데이터를 불러올 수 있습니다:
```
한국어에서 쌍점으로 문장을 끝마치는 경우는 적은 것 같아요. 온점으로 대체를 고려해주세요!
해당하는 라인은:
- [ ] L30
- [ ] L48
- [ ] L66
- [ ] L72
- [ ] L82
- [ ] L134
- [ ] L165
- [ ] L227
- [ ] L282
- [ ] L292
- [ ] L307
- [ ] L332
- [ ] L359
- [ ] L369
- [ ] L398
- [ ] L417
- [ ] L483
- [ ] L489
- [ ] L508
- [ ] L517
- [ ] L528

wonhyeongseo reviewed (changes requested):
I loved reading your translation. See you soon, @sim-so !

gabrielwithappy reviewed (approved):
It looks good to me! :-) 👍 
I think you can ask review to 🤗 members.

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -0,0 +1,535 @@
+이미지 전처리는 종종 몇 가지 이미지 증강 다음에 올 수도 있습니다.
+이미지 전처리 및 이미지 증강은 모두 이미지 데이터를 변형하지만, 서로 다른 목적을 가지고 있습니다:
+
+* 이미지 증강은 과적합을 방지하고 모델의 견고성을 높이는 데 도움이 되는 방식으로 이미지를 수정합니다. 
```
확인이 늦어서 죄송합니다!
저도 주요 단어들은 영문 표기를 병기하는 게 좋은 것 같은데요, 앞으로 어떤 경우에 영문 표기를 붙이면 좋을까요?

sim-so commented on docs/source/ko/preprocessing.mdx:
````diff
@@ -0,0 +1,535 @@
+(100000,)
+```
+
+## 컴퓨터비전[[computer-vision]]
````
```suggestion
## 컴퓨터 비전[[computer-vision]]
```
띄어쓰는 것으로 수정합니다.

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -0,0 +1,535 @@
+다른 데이터 증강 라이브러리를 사용하는 방법이 알고 싶다면, [Albumentations](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification_albumentations.ipynb) 또는 [Kornia notebooks](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification_kornia.ipynb)에서 배울 수 있습니다.
+
+1. [`Compose`](https://pytorch.org/vision/master/generated/torchvision.transforms.Compose.html)를 사용하여  [`RandomResizedCrop`](https://pytorch.org/vision/main/generated/torchvision.transforms.RandomResizedCrop.html) 및 [`ColorJitter`](https://pytorch.org/vision/main/generated/torchvision.transforms.ColorJitter.html)와 같은 몇 가지 변환을 연결합니다.
+참고로, 크기 조정에 대해서는 `image_processor`에서 이미지 크기 요구사항을 가져올 수 있습니다. 
```
```suggestion
참고로 크기 조정에 필요한 이미지의 크기 요구사항은 `image_processor`에서 가져올 수 있습니다. 
```
제안 내용 참고하여 위와 같이 수정하고자 합니다!

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so commented on docs/source/ko/preprocessing.mdx:
````diff
@@ -497,21 +499,22 @@ Now take a look at the `audio` and `text` columns:
 ```
 
-Remember you should always [resample](preprocessing#audio) your audio dataset's sampling rate to match the sampling rate of the dataset used to pretrain a model!
+기존에 사전 학습된 모델에서 사용된 데이터셋과 새로운 오디오 데이터셋의 샘플링 레이트를 일치시키기 위해 오디오 데이터셋의 샘플링 레이트를 [리샘플링](preprocessing#audio)해야 합니다!
````
I will replace all "sampling rate" to "샘플링 레이트". Thank you for your detailed review!

sim-so commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -0,0 +1,535 @@
+* `sampling_rate`는 음성 신호에서 초당 측정되는 데이터 포인트 수를 나타냅니다.
+
+이 튜토리얼에서는 [Wav2Vec2](https://huggingface.co/facebook/wav2vec2-base) 모델을 사용합니다. 모델 카드를 보면 Wav2Vec2가 16kHz 샘플링된 음성 오디오를 기반으로 사전 학습된 것을 알 수 있습니다. 
+모델을 사전 학습하는 데 사용된 데이터셋의 샘플링 속도와 오디오 데이터의 샘플링 속도가 일치해야 합니다. 데이터의 샘플링 속도가 다르면 데이터를 리샘플링해야 합니다.
```
```suggestion
모델을 사전 학습하는 데 사용된 데이터셋의 샘플링 레이트와 오디오 데이터의 샘플링 레이트가 일치해야 합니다. 데이터의 샘플링 레이트가 다르면 데이터를 리샘플링해야 합니다.
```

sim-so commented on docs/source/ko/preprocessing.mdx:
```diff
@@ -0,0 +1,535 @@
+이 튜토리얼에서는 [Wav2Vec2](https://huggingface.co/facebook/wav2vec2-base) 모델을 사용합니다. 모델 카드를 보면 Wav2Vec2가 16kHz 샘플링된 음성 오디오를 기반으로 사전 학습된 것을 알 수 있습니다. 
+모델을 사전 학습하는 데 사용된 데이터셋의 샘플링 속도와 오디오 데이터의 샘플링 속도가 일치해야 합니다. 데이터의 샘플링 속도가 다르면 데이터를 리샘플링해야 합니다.
+
+1. 🤗 Datasets의 [`~datasets.Dataset.cast_column`] 메서드를 사용하여 샘플링 속도를 16kHz로 업샘플링하세요:
```
```suggestion
1. 🤗 Datasets의 [`~datasets.Dataset.cast_column`] 메서드를 사용하여 샘플링 레이트를 16kHz로 업샘플링하세요:
```

sim-so committed:
Update docs/source/ko/preprocessing.mdx

Co-authored-by: Wonhyeong Seo <wonhseo@kakao.com>

sim-so reviewed (commented):
Thanks to your reviews, I fixed several lines. :smile:

sim-so committed:
Update docs/source/ko/preprocessing.mdx

sim-so committed:
Update docs/source/ko/preprocessing.mdx

sim-so committed:
Update docs/source/ko/preprocessing.mdx

sim-so committed:
Update docs/source/ko/preprocessing.mdx

sim-so committed:
Update docs/source/ko/preprocessing.mdx

sim-so committed:
Update docs/source/ko/preprocessing.mdx

sim-so committed:
Merge branch 'huggingface:main' into ko-preprocessing

sim-so committed:
fixed translation

sim-so commented:
다른 문서들을 참고하여 일부 단어의 번역어를 모두 변경했습니다.
- argument -> 인수
- feature extractor -> 특성 추출기
- method -> 메소드
- separator([SEP]) -> 분할 토큰

sim-so committed:
Merge branch 'main' into ko-preprocessing

sim-so commented:
Could you review this PR?
@sgugger, @ArthurZucker, @eunseojo

sgugger reviewed (approved):
Thanks a lot for your contribution!
